DataFormats/Math/interface/alpaka/deltaPhi.h

```diff
@@ -0,0 +1,35 @@
+#ifndef DataFormats_Math_alpaka_deltaPhi_h
+#define DataFormats_Math_alpaka_deltaPhi_h
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+
+namespace cms::alpakatools {
+
+  // reduce to [-pi,pi]
+  template <typename TAcc, typename T>
+  ALPAKA_FN_HOST_ACC ALPAKA_FN_INLINE constexpr T reduceRange(TAcc const& acc, T x) {
+    constexpr T o2pi = 1. / (2. * M_PI);
```

Comment:
Since CMSSW now uses C++20 you could use `std::numbers::pi_v<T>` or `std::numbers::inv_pi_v<T>`